cloud/google/clusteractuator.go

```diff
@@ -186,6 +209,21 @@ func (gce *GCEClusterClient) deleteFirewallRule(cluster *clusterv1.Cluster, rule
 	return gce.computeService.WaitForOperation(clusterConfig.Project, op)
 }
 
+func (gce *GCEClusterClient) handleClusterError(cluster *clusterv1.Cluster, err *apierrors.ClusterError, eventAction string) error {
```

Comment:
should this be named just handleError, as the cluster is redundant in this file?

Reply:
updated